gh-deployments-environments/action.yaml

```diff
@@ -0,0 +1,77 @@
+---
+name: GH Deployments and Environments
+description: |
+  Creates, updates and deletes GitHub Deployments and Environments
+inputs:
+  action:
+    description: One of create-deployment, create-deployment-status or delete-environment
+    required: true
```

Comment:
If it has to be one of three options maybe consider using `type: choice` for that particular input.

Reply:
Action inputs do not have a `type`, they are always strings: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
`type` only works for workflows.

Reply:
Are you sure about that? :smiley: 
https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/actions/slack-notification/action.yml#L9

Reply:
I know about that one.  Have we ever tested what's going to happen when you use input which actually has a value that is not listed as a choice?  I'll follow docs on this one but I'm going to test that thing.

Reply:
I'll resolve this conversation, it's approved and I'd like to use that tiny github action already in one tiny workflow we have that tidies up featurenets.